src/addons/proxymonitor/common/src/main/resources/monitoring.xsd

```diff
@@ -61,8 +61,30 @@
             </xs:sequence>
         </xs:complexType>
     </xs:element>
+
+
+    <xs:complexType name="OutputSpecType">
+        <xs:sequence>
+            <xs:element name="outputField" type="xs:string" minOccurs="0" maxOccurs="unbounded">
+                <xs:annotation>
+                    <xs:documentation>Name of the operational data field</xs:documentation>
+                </xs:annotation>
+            </xs:element>
+        </xs:sequence>
+    </xs:complexType>
+
     <xs:complexType name="getSecurityServerMetricsType">
-        <xs:sequence/>
+        <xs:sequence>
+            <xs:element name="outputSpec" type="tns:OutputSpecType">
```

Comment:
Should it have `minOccurs="0" maxOccurs="unbounded"` as in operational monitoring's `OutputSpecType`?

Reply:
outputSpec was missing min/maxOccurs. Corrected.